<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>askBaseProbability</class>
 <widget class="QDialog" name="askBaseProbability">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="chooseBase">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Base Probability of 0.25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Base Probability calculated from the input sequence</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Custom Base Probability</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="spinA">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinC">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinG">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinT">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>66</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editFileName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name of the matrix file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Matrix as:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseMatrix">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Standard Probability Matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Normalized Probability Matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Standard Logarithmic Matrix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Normalized Logarithmic Matrix</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>283</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please give a length:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="intPosition">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>253</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Please give a position:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="intLength">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>283</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
